packages/browser/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+export function resolveAsyncGlobal(name: string): Promise<any> {
```

Comment:
usually I'd say its better to write a simple unit test for this. but I guess you probably had this tested on local already

Reply:
I did think about it, but setting up a timing-dependent test for this simple function didn't seem worth it.